src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/AcknowledgeServiceCallbackHandler.java

```diff
@@ -40,12 +43,12 @@
     private final WorkingDayIndicator workingDayIndicator;
 
     @Override
-    protected Map<CallbackType, Callback> callbacks() {
+    protected Map<String, Callback> callbacks() {
         return Map.of(
-            CallbackType.ABOUT_TO_START, this::emptyCallbackResponse,
-            CallbackType.MID, this::validateDateOfBirth,
-            CallbackType.ABOUT_TO_SUBMIT, this::setNewResponseDeadline,
-            CallbackType.SUBMITTED, this::buildConfirmation
+            callbackKey(ABOUT_TO_START), this::emptyCallbackResponse,
+            callbackKey(MID, "confirm-details"), this::validateDateOfBirth,
```

Comment:
how about keeping all these page ids in an enum?

Reply:
enum is not going to add any value here so will keep string.